main/test/flix/Test.Type.Bool.flix

```diff
@@ -0,0 +1,263 @@
+
+enum B[_: Bool] with Eq {
+    case A
+}
+
+def assertEqual(_: B[a], _: B[a]): Bool = true
+def assertUnequal(_ : B[a], _ : B[not a]): Bool = true
+def assertTrue(_: B[true]): Bool = true
+def assertFalse(_: B[false]): Bool = true
+def assertXor(_: B[a], _: B[not a]): Bool = true
```

Comment:
Already have this, so can delete the other.

Reply:
Good heavens, that somehow never crossed my mind...